modules/internal/ExportWrappers.chpl

```diff
@@ -24,6 +24,31 @@ module ExportWrappers {
   use CPtr;
   use SysCTypes;
 
+  private proc _initDynamicEndCount() {
+    var endCount = _endCountAlloc(false);
+    __primitive('set dynamic end count', endCount);
+    return;
```

Comment:
The `return` in these 4 procs seems superfluous.

Reply:
That's a bad personal preference.